gui.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         初始化
         """
         try:
+            self.config = MyConfig()
```

Comment:
I don't think you need to put config in self.

Reply:
If not done so, the following error will be encountered. Perhaps you have better suggestions for modifications. This part is up to you to decide.

```
*cpt["config"], is_half=MyConfig().is_half
NameError: name 'config' is not defined
```

Reply:
No, you're right.
It have a class with the same name.
Thank you for the correction.

Reply:
Well, why we have to rename `Config` class to `MyConfig` ? 😂

Reply:
I think that would be best.

Reply:
@fumiama Shall I do it?

Reply:
> @fumiama Shall I do it?

Sure. You can edit this PR directly.

Reply:
@fumiama
I made some changes that are not included in this PR, so I created another PR #340
I'd like to see #340 merged after this PR is merged